pkg/storage/mongo/getter.go

```diff
@@ -16,26 +16,11 @@ import (
 // Info implements storage.Getter
 func (s *ModuleStore) Info(ctx context.Context, module, vsn string) ([]byte, error) {
 	const op errors.Op = "mongo.Info"
-	ctx, span := observ.StartSpan(ctx, op.String())
```

Comment:
Can you keep this span here to collect traces?

Reply:
No problem - Should I also start a trace for the new function (query) or will the trace on Info capture the desired metrics?

Reply:
yes! that is super important imo as that is where we will do majority of compute time. By having traces here, we can see which function the query was a subset of

```
----Info----- ------GoMod------
 --query---     ----query---
```